UI/assets/js/main.js

```diff
@@ -1,111 +1,94 @@
+const commentContainer = document.querySelector('.comment-div');
 
 let toggleState = 0;
-let voteCheck =0;
-let upVoteCount =1;
-let downVoteCount =1;
-// toggle display of question input form
-questionBtn.onclick = function () {
-  if (toggleState == 0) {
-    askQuestionContainer.classList.remove("d-none")
-    askQuestionContainer.classList.add("d-flex")
-    toggleState = 1;
-
-  } else {
-    askQuestionContainer.classList.add("d-none")
-    askQuestionContainer.classList.remove("d-flex")
-    toggleState = 0;
-  }
-}
+// let voteCheck = 0;
+// let upVoteCount = 1;
+// let downVoteCount = 1;
 
 // Toggle rsvp user effect(color)
-rsvpBtn.onclick = function () {
-  if (toggleState == 0) {
-    rsvpIcon.classList.remove("text-grey")
-    rsvpIcon.classList.add("text-blue")
-    document.querySelector(".rsvp-tooltip-text").innerHTML = "Click here to cancel";
+const rsvp = () => {
+  if (toggleState === 0) {
+    rsvpIcon.classList.remove('text-grey');
+    rsvpIcon.classList.add('text-blue');
+    document.querySelector('.rsvp-tooltip-text').innerHTML = 'Click here to cancel';
     toggleState = 1;
-
   } else {
-    rsvpIcon.classList.add("text-grey")
-    rsvpIcon.classList.remove("text-blue")
-    document.querySelector(".rsvp-tooltip-text").innerHTML = "Click here to rsvp";
+    rsvpIcon.classList.add('text-grey');
+    rsvpIcon.classList.remove('text-blue');
+    document.querySelector('.rsvp-tooltip-text').innerHTML = 'Click here to rsvp';
     toggleState = 0;
   }
-}
+};
 
-// upvote funtion
-function upVote() {
-  upVoteAmount = Number(upVoteAmount);
-  voteCheck = upVoteAmount + 1;
-  if (voteCheck%2 !== 0) {
-    upVoteAmount += 1;
-    let newUpVoteAmount = document.querySelector(".up-vote-amount");
-    newUpVoteAmount.textContent = upVoteAmount;
-    upVoteBtn.classList.add("text-blue")
-  } else if (voteCheck%2 === 0) {
-    upVoteAmount -= 1;
-    let newUpVoteAmount = document.querySelector(".up-vote-amount");
-    newUpVoteAmount.textContent = upVoteAmount;
-    upVoteBtn.classList.remove("text-blue")
-  }
-}
-upVoteBtn.onclick = function () {
-
-  if (downVoteCount%2 !== 0) {
-    upVote();
-  } else {
-    return
-  }
-
-  upVoteCount ++;
-}
-
-// downvote funtion
-function downVote() {
-    downVoteAmount = Number(downVoteAmount);
-    voteCheck = downVoteAmount + 1;
-    if (voteCheck%2 !== 0) {
-      downVoteAmount += 1;
-      let newDownVoteAmount = document.querySelector(".down-vote-amount");
-      newDownVoteAmount.textContent = downVoteAmount;
-      downVoteBtn.classList.add("text-red")
-    } else if (voteCheck%2 === 0) {
-      downVoteAmount -= 1;
-      let newDownVoteAmount = document.querySelector(".down-vote-amount");
-      newDownVoteAmount.textContent = downVoteAmount;
-      downVoteBtn.classList.remove("text-red")
-    }
-}
-downVoteBtn.onclick = function () {
-  if (upVoteCount%2 !== 0) {
-    downVote();
-  } else {
-    return
-  }
-  downVoteCount ++;
-}
+// // upvote funtion
```

Comment:
Why was this function commented out?

Reply:
This was an oversight, while trying to refactor the rsvp function to ES6, it will be un-commented in the next PR.

Reply:
You should be able to do that in this PR. Pushing a new commit will reflect on this PR

Reply:
Okay, I will be doing that right away

Reply:
It has been updated in new commit